bin/common/usr_local_bin_to_path.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# Add /usr/local/bin to non-login path, since tools are installed there
+if ! grep ENV_SUPATH /etc/login.defs | grep -q /usr/local/bin; then
+  sed -i '/ENV_SUPATH/s/$/:\/usr\/local\/bin/' /etc/login.defs
```

Comment:
If you take a look at https://github.com/SUSE/scf/blob/d96c25ea87d6e7a0838e0d9aa718566e50215df5/packer/Vagrantfile.template#L40-L43, I'm running [create_k8s_certs.sh](https://github.com/SUSE/scf/blob/d96c25ea87d6e7a0838e0d9aa718566e50215df5/bin/common/create_k8s_certs.sh) right after this. Certstrap (and everything else now) is getting installed to `/usr/local/bin`, but `/usr/local/bin` doesn't go in the path by default when run via a non-login shell (like when using the shell provisioners). The alternatives were putting these utilities in one of `/usr/bin:/bin:/usr/games` (the paths common to both vagrant and root non-login shells), all of which seemed less appropriate